lib/ufe/UsdUndoRenameCommand.cpp

```diff
@@ -62,32 +64,56 @@ UsdSceneItem::Ptr UsdUndoRenameCommand::renamedItem() const
 	return fUfeDstItem;
 }
 
-bool UsdUndoRenameCommand::rename(SdfLayerHandle layer, const Ufe::Path& ufeSrcPath, const SdfPath& usdSrcPath, const SdfPath& usdDstPath)
+bool UsdUndoRenameCommand::renameRedo()
```

Comment:
Re-implemented USD renaming.  We now SdfCopySpec the source to the destination, in the same layer, and inactivate the source, on do / redo.  On undo, we RemovePrim the destination, and activate the source.  This is arguably a naïve implementation, but it's a start.

Unfortunately, undo fails, because RemovePrim fails, not clear why at this point, will continue investigation.

Reply:
Thanks to @mattyjams , undo / redo works!